pkg/snapstore/ocs_s3_snapstore.go

```diff
@@ -177,17 +179,50 @@ func isOCSConfigEmpty(config ocsAuthOptions) error {
 	return fmt.Errorf("ocs s3 credentials: region, secretAccessKey, endpoint or accessKeyID is missing")
 }
 
-// OCSSnapStoreHash calculates and returns the hash of OCS snapstore secret.
-func OCSSnapStoreHash(config *brtypes.SnapstoreConfig) (string, error) {
-	ao, err := getOCSAuthOptions("")
-	if err != nil {
-		return "", err
+// OCSSnapStoreModTime returns the lastest modification timestamp of the OCS credential file(s)
+func OCSSnapStoreModTime(config *brtypes.SnapstoreConfig) (time.Time, error) {
```

Comment:
```suggestion
func OCSSnapStoreModTime() (time.Time, error) {
```
same as above, and all providers as well